backend/src/main/kotlin/app/ehrenamtskarte/backend/migration/migrations/V0014_CreateUserEntitlementsTable.kt

```diff
@@ -17,7 +17,7 @@ internal class V0014_CreateUserEntitlementsTable : Migration() {
                 "startDate" date NOT NULL,
                 "endDate" date NOT NULL,
                 "revoked" boolean NOT NULL,
-                "regionId" integer NOT NULL,
```

Comment:
oh looks like we had a misunderstanding. You can keep here the `regionId` we just use the `regionKey` to identify the correct region (id) from the regions table @seluianova

Reply:
Fixed 🙈